tests/test_random.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from stellargraph.random import SeededPerBatch
+import numpy as np
+
+
+def test_seeded_per_batch():
+    num_batches = 10
+    num_iter = 10
+    seed = 0
+
+    # different permutations of batch numbers
```

Comment:
It could be a bit hard to reverse engineer the purpose of this test in weeks/months, so maybe this comment could be expanded slightly (or another one added)? e.g.:

```suggestion
    # different permutations of batch numbers should always give the same seeds
```